.devcontainer/docker-compose.yml

```diff
@@ -29,9 +29,6 @@ services:
     environment:
       - MONGO_URL=mongodb://wekandb-dev:27017/wekan
       - ROOT_URL=http://localhost:3000
-      #- MAIL_URL=smtp://user:pass@mailserver.example.com:25/
```

Comment:
Please add back mail settings. Mail also is tested in development.

Reply:
Of course it is. But there are some problems, when the variable is set in the `docker-compose.yml` which is in source code management.

- When I insert my data here, the file is modified and git sees it as changed and I have to be careful not to insert my personal data here.
- As soon as the variable is set (even with demodata) wekan tries so send the mail which gives an exception in the docker console, but not the mailoutput. So in development ist useful to see the mail, without sending it.

There is a way to set the variables without getting into the risk to commit personal data:

- We add the reference to an `.devcontainer/docker-compose.extend.yml` file to the `devcontainer.json`
- We add the extend-file to `.gitignore` so that nobody exposes personal data to github.
- An empty extend file only needs `version : '3.7'`

If someone needs to modify or overwrite the minimal-default config, he can add the required data to the extend file, which modifies the `docker-compose.yml`:

```yml
version: '3.7'
services:
  wekan-dev:
    # ADD Custom vars out of git
    environment:
      - MAIL_URL=smtp://user:pass@mailserver.example.com:25/
      - MAIL_URL=smtp://<mail_url>:25/?ignoreTLS=true&tls={rejectUnauthorized:false}
      - MAIL_FROM=Wekan Notifications <noreply.wekan@mydomain.com>
```